conda/meta.yaml

```diff
@@ -1,11 +1,11 @@
 package:
     name: odl
-    version: "0.6.1"
+    version: "1.0.0"
```

Comment:
Dev?

Reply:
Hm, at the time I didn't use it, maybe because it wasn't supported? But it makes sense for uploading dev releases to a non-official channel, or for local testing. I'll add it.